src/main/webapp/WEB-INF/views/reservation-form.jsp

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<!--
+    D.Bonis
+    09/05/23
+
+    Reservation Requesting Page
+    Update - 09/7/23 -- Added comments, added toHome, added roomPrice with logic
+    Update - 09/8/23 -- Made it clear previous fill in on page load, added springboot
+                        specific form tag, added total value in hidden attribute, now saves to database no problem
+-->
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<html lang="en">
+<head>
+    <title>Reservation Form</title>
+    <link rel="stylesheet" type="text/css" href="css/reservationFormStyle.css">
+</head>
+<body>
+    <header>
+		<a href="home"><img id=logo src="_images/MB_Title_Logo_3.png"></img></a>
+		<!-- USER ICON-->
+		<a href="login"><i class="fa fa-user-circle-o" style="float: right; font-size: 56px; color: #554D41"></i></a>
+	</header>
+
+
+     <!-- Form that lets you fill out roomSize, numberOfGuests, checkInDate, and checkOut Date
+             ReservationId Is Generated, UserId is used from login, and total price is determined by number of guests and days-->
+    <form:form action="/reservation" method="POST" id="reservationForm">
+    <div class="fullForm">
+		<h1>Reservation Form</h1>
+		
+            <div class="formTop">
+                <div class="picker">
+                    <h2>Check In Date:</h2> <br>
+                    <input type="date" id="checkInDate" name="checkInDate" required></input>
+                </div>
+                
+                <div class="picker">
+                    <h2>Check Out Date:</h2> <br>
+                    <input type="date" id="checkOutDate" name="checkOutDate" required></input>
+                </div>
+                
+                <div class="picker">
+                    <h2>Number Of Guests:</h2> <br>
+                    <select id="numberOfGuests" name="numberOfGuests required">
```

Comment:
We need to close the String quote. select id="numberOfGuests" name="numberOfGuests" required